examples/swift/README.md

```diff
@@ -0,0 +1,26 @@
+## Swift examples
+
+### Permissions
+
+Before running the examples, you have to add execute permissions.
+
+```
+$ chmod +x count.swift
+$ chmod +x greeter.swift
```

Comment:
Oh, I didn't know that, thanks! It looks like, I already had the files checked into github with +x.
I'll remove the permissions part.
You never stop learning :+1: